README.md

```diff
@@ -87,59 +89,51 @@ Known good configuration for EC2 instance:
        "deployDataCatalog": <false || true deploy data catalog>
    }
    ```
-   
-4. Export your dev account number and deployment username:
 
-   ```
-   export ACCOUNT_NUMBER=<target account number>
-   export AWS_DEFAULT_REGION=<target region for deployment>
-   export NAME=<unique name for stacks>
-   ```
+4. Optional: If you want to enable Authentication, please head over to [Enabling Authentication](#enabling-authentication) in this README.
 
-5. Optional: If you want to enable Authentication, please head over to [Enabling Authentication](#enabling-authentication) in this README.
-
-6. Optional, export the following environment variable if you wish to build your containers from source using the submodules.
+5. Optional, export the following environment variable if you wish to build your containers from source using the submodules.
 
    ```
    export BUILD_FROM_SOURCE=true
    ```
 
-7. Pull your latest credentials into `~/.aws/credentials`
+6. Pull your latest credentials into `~/.aws/credentials`
```

Comment:
if you prefix all of these with `1.` it will automatically increment the list for you. more maintainable.

Reply:
updated